install-requirements.sh

```diff
@@ -2,7 +2,15 @@
 
 set -e
 
+# install dependencies
 for file in `ls functions/*/requirements.txt`
 do
   pip install --upgrade -r $file -t `dirname $file` 
 done
+
+# copy shared tools
+pip install --upgrade -r tools/requirements.txt -t tools
+for dir in `ls -d functions/*/`
+do
+  cp -R tools $dir
```

Comment:
@neutrinog @richmahn - it looks like there needs to be some changes to project.master.json to support build hook.